client/sites-dashboard/components/sites-site-plan.tsx

```diff
@@ -1,3 +1,4 @@
+import { PLAN_ECOMMERCE_TRIAL_MONTHLY } from '@automattic/calypso-products';
```

Comment:
This may be a good opportunity for us to create an `isECommerceTrial()` helper function in `calypso-products`, but that can also be something we punt on.

Reply:
There is an [isSiteOnECommerceTrial](https://github.com/Automattic/wp-calypso/blob/trunk/client/state/sites/plans/selectors/is-site-on-ecommerce-trial.ts) function already but it doesn't work in this context.

I was worried it would be confusing to add another, similar, function. But we can revisit another time.